lxd/api_internal.go

```diff
@@ -788,6 +789,8 @@ func internalImportFromBackup(d *Daemon, projectName string, instName string, fo
 		return err
 	}
 
+	instDBArgs.Config["volatile.uuid"] = uuid.New()
```

Comment:
We'll be wanting a comment (similar to what you had before, but with the changes I suggested) explaining *why* this is here.

Reply:
Just a nit, but missing full stop.